shadows/framework/src/main/java/org/robolectric/shadows/ShadowActivity.java

```diff
@@ -571,6 +576,12 @@ public int getPendingTransitionExitAnimationResourceId() {
     return pendingTransitionExitAnimResId;
   }
 
+  @Nullable
+  public OverriddenActivityTransition getOverriddenActivityTransition(
```

Comment:
Added `@RequiresApi(Build.VERSION_CODES.UPSIDE_DOWN_CAKE)` for this getter method only. I suppose we don't need the same annotation for the new field in `ShadowActivity`.

Reply:
I updated one of the old tests in `NormalCompatibilityTest`, which created a `TestActivity`. Now it also creates a shadow of the activity and is wrapped in Kotlin `use` to prevent the memory leak from a simple `Robolectric.setupActivity()` call.

Those compat-28 tests all fail for me locally, so I'm not sure if I did it correctly. Take a look please if that's what you meant.